packages/terra-notification-dialog/tests/jest/Prompt.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Many of these test files are named `Prompt`. I saw on the tech design and branch name this component was called `Prompt` but the code and package name look like they've been changed to `terra-notification-dialog`. I would update all occurrences of the conflicting names to match the desired package name.